src/Widgets/BaseIcon.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
@yemexx1 The use for this widget has been replaced with a static function in the `CottaCush\Yii2\Helpers\Html` class.